templates/web/base/maps/_map_instructions.html

```diff
@@ -0,0 +1,20 @@
+<div class="map-instructions" aria-live="polite">
+    <div class="keyboard-intructions-wrapper" id="keyboard-instructions-first">
+        <span>[% loc("You can move around with your keyboard") %]</span>
+        <div class="keyboard-key-group">
+            <div class="keyboard-key-icon zoom-in">[% loc("plus") %]</div>
```

Comment:
Sorry, just noticed - this should be "zoom in" - the text should be an alternative, not a description.